13-tool-usage-cycle-tracking/README.md

```diff
@@ -316,6 +390,8 @@ The event body also includes these named values:
 * `apparentPower`: The measured apparent power (in VA).
 * `powerFactor`: The power factor - active power divided by apparent power.
 * `active`: set to True to indicate the corresponding instance pin is active.
+* `vibration_raw`: the amount of vibration sensed
+* `vibration`: the vibration category (when [vibration thresholds](#vibration-monitoring-and-alerts) are configured.)
```

Comment:
The vibration category (when [vibration thresholds](#vibration-monitoring-and-alerts) are configured).